front/app/containers/ProjectsShowPage/shared/document_annotation/Konveio.tsx

```diff
@@ -32,8 +32,12 @@ const Konveio = ({ documentUrl, className }: Props) => {
   urlParams['integration'] = 'CitizenLab';
   urlParams['iframe'] = 'true';
 
-  if (email !== null) {
-    urlParams['username'] = email;
+  if (typeof email === 'string') {
```

Comment:
`typeof` is safer than `!==`. I changed `email` in this file. Now it can be `undefined`, but this would pass.